acceptance/setup/pre_suite/50_install_modules.rb

```diff
@@ -4,4 +4,8 @@
 
 step "Install the puppetdb module and dependencies" do
   on master, "puppet module install puppetlabs/puppetdb"
+  if options[:type] == 'aio'
+    # patch puppetdb module for version 4.1.0
```

Comment:
@kbarber this has been pulled into your scrum team. I assigned it to @ajroetker . https://tickets.puppetlabs.com/browse/MODULES-1905